### DISCUSSION TITLE
Larger Signs

### DISCUSSION THREAD

**Discussion by Scwestbrook** [ChAoSUnItY/EkiLib#4] [Ideas]
Hi! I really love this mod and want it to be the best it can be. So with that in mind I've thought up an idea (or is it a bug?) where if I place a station sign, and then another next to it, they would merge and form kind of a large station sign for the case that your station name was too long. I'd also like you to make it where the sign can be broken on any block, not just the center, and also for them to break if the block behind them breaks.

**Comment by ChAoSUnItY** [comment]
For the first suggestion, it's a good idea, after several checks, it's possible to do it, I'll try add this feature in 1.1.0.

But for the next idea, I'm not quite sure what's the main idea, can you give me much clearer describe? Thanks.

**Reply by Scwestbrook** [reply]
For the second idea, right now, you can only break it from the block it was placed on, but what if I could break it by breaking the sign on the left or right. Could you also make it so that when the block behind it is broken, the sign breaks, to avoid floating signs?
